staging/src/k8s.io/csi-translation-lib/plugins/in_tree_volume.go

```diff
@@ -165,6 +219,57 @@ func translateTopology(pv *v1.PersistentVolume, topologyKey string) error {
 	return nil
 }
 
+// translateTopologyFromCSIToInTree
+// 1. Remove any existing CSI topologyKey from NodeAffinity
+// 2. Add Kubernetes Topology in the NodeAffinity if it does not exist
+func translateTopologyFromCSIToInTree(pv *v1.PersistentVolume, topologyKey string) error {
+
+	csiTopologyZoneValues := getTopologyValues(pv, topologyKey)
+
+	// Frist remove the CSI topology Key
```

Comment:
instead of remove/add, can we use replaceTopology?

Reply:
This function assumes that we always have one CSI Topology label in the NodeAffinity. But we need to supply both zone/region labels. So I think a remove and adding new labels seems more clear what we are trying to achieve.